simln-lib/src/test_utils.rs

```diff
@@ -212,6 +212,7 @@ pub fn create_activity(
     amount_msat: u64,
 ) -> ActivityDefinition {
     ActivityDefinition {
+        name: None,
```

Comment:
Commit message doesn't match the code - we're not adding `name` to `ActivityDefinition` here, just setting it